### PR TITLE
Standalone publisher editorial plugins interfering

### LINKED ISSUE #579: Publish EDL
<div> Freshdesk Ticket ID: <a href="https://pype.freshdesk.com/a/tickets/258">258</a></div>
                       <div> Freshdesk Ticket Agent: Jakub Jezek</div>
                       <div> Freshdesk Ticket Agent Email: jakub@pype.club</div>
                       <div> Ticket Priority: High</div>
                       <div> Freshdesk Ticket Description: </div><div style="font-family:-apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Arial, sans-serif; font-size:14px">
<div dir="ltr">We need to test EDL publishing and make sure it doesn't clash with other families.</div>
<div dir="ltr"><br></div>
<div dir="ltr"><br></div>
</div>

### PR DESCRIPTION
**Pull request by jezscha**
- editorial plugins were interfering with other families
- adding "clip" family to better filter editorial instances 

- render/plates workflow fixes including: 
    - space in input path was crashing standalone publisher
    - space in input path was crashing burning ocio script
    - when input resolution was not dividable by two it was crashing ffmpeg (solution was to add pad filter to add half frame pad to both sides)

closes #579

### CODE REVIEW

**Comment by jezscha** [comment]
@mkolar it is check on client project and working well now

**Comment by jezscha** [comment]
@mkolar i was commiting from client machine with your user on it.. sry

File: pype/plugins/global/publish/extract_review.py [hunk 1]
```diff
@@ -453,6 +453,7 @@ def ffmpeg_full_args(
         if audio_filters:
             all_args.append("-filter:a {}".format(",".join(audio_filters)))
 
+        all_args.append('-vf pad="width=ceil(iw/2)*2:height=ceil(ih/2)*2"')
```
Comment (iLLiCiTiT) [inline pype/plugins/global/publish/extract_review.py:456]: This will break what is happening in `def rescaling_filters`.

Comment (jezscha) [reply pype/plugins/global/publish/extract_review.py:456]: any idea how to fix it the way it will not break those filters?

**Pull request merged**